<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThrusterSettings</class>
 <widget class="QWidget" name="ThrusterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="thrustersVerticalLayout"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxVelosity">
       <property name="title">
        <string>Velocity</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QSlider" name="verticalSliderVelocity">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxVelocity">
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxForwardK">
       <property name="title">
        <string>Forward K</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QSlider" name="verticalSliderForwardK">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBoxForwardK">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxBackwardK">
       <property name="title">
        <string>Backward K</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_23">
        <item>
         <widget class="QSlider" name="verticalSliderBackwardK">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBoxBackwardK">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxForwardSaturation">
       <property name="title">
        <string>Forward Saturation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_24">
        <item>
         <widget class="QSlider" name="verticalSliderForwardSaturation">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxForwardSaturation">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxBackwardSaturation">
       <property name="title">
        <string>Backward Saturation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_25">
        <item>
         <widget class="QSlider" name="verticalSliderBarkwardSaturation">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxBarkwardSaturation">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
